packages/components/src/atoms/Link/Link.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react'
+import type { ElementType, ReactElement } from 'react'
+import { PolymorphicComponentPropsWithRef, PolymorphicRef } from '../../typings'
+
+type LinkBaseProps = {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+ * Specifies the component variant.
+ */
+  variant?: 'default' | 'display' | 'footer' | 'inline'
+  /**
+  * Defines use of inverted color.
+  */
+  inverse?: boolean
+  /**
+   * Defines size os element
+   */
+  size?: 'small' | 'regular'
+}
+
+export type LinkElementType = ElementType
```

Comment:
Do we have a specific case that we gonna need this type?

Reply:
I had problems with different versions of react, this type can fix it